src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -0,0 +1,754 @@
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using System.Management.Automation;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.ResourceManager;
+using System.Text.RegularExpressions;
+using System.IO;
+using System.Text;
+using System.Runtime.InteropServices;
+using System;
+using System.Security.Cryptography;
+using System.Diagnostics;
+using System.Net;
+using Microsoft.Azure.Commands.Common.Exceptions;
+using System.Collections.Generic;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Authentication.Factories;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions.Models;
+using Microsoft.Azure.PowerShell.Cmdlets.Ssh.Common;
+using Newtonsoft.Json;
+using Azure.ResourceManager.HybridConnectivity.Models;
+using Microsoft.Rest.Azure;
+using Newtonsoft.Json.Linq;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+
+namespace Microsoft.Azure.Commands.Ssh
+{
+    public abstract class SshBaseCmdlet : AzureRMCmdlet
+    {
+
+        #region Constants
+        public const string clientProxyStorageUrl = "https://sshproxysa.blob.core.windows.net";
+        public const string clientProxyRelease = "release01-11-21";
+        public const string clientProxyVersion = "1.3.017634";
+
+        public const string InteractiveParameterSet = "Interactive";
+        public const string ResourceIdParameterSet = "ResourceId";
+        public const string IpAddressParameterSet = "IpAddress";
+        #endregion
+
+        public static readonly string [] supportedResourceTypes = {"Microsoft.Compute/virtualMachines", "Microsoft.HybridCompute/machines" };
+        
+        protected bool deleteKeys;
+        protected bool deleteCert;
+        
+        public string proxyPath;
+        public string relayInfo;
+        public string aadCertificate;
+        protected DateTime relayInfoExpiration;
+        private RMProfileClient profileClient;
+        private SshAzureUtils azureUtils;
+
+        public SshAzureUtils AzureUtils
```

Comment:
Generally, cmdlet class properties that are not PowerShell parameters, are `internal` access.

Reply:
Changed the properties to internal.